test/rgb.collection.js

```diff
@@ -291,21 +291,36 @@ exports["RGB.Collection"] = {
 
 
   "Animation.render": function(test) {
-    test.expect(1);
+    test.expect(4);
```

Comment:
conceptually, what are you expecting to equal 4 here? Are there any docs or other guidance for understanding these test cases?

Reply:
Johnny-Five tests are written with [nodeunit](https://github.com/caolan/nodeunit).  API docs are [inline in that README, here](https://github.com/caolan/nodeunit#api-documentation).  `expect(count)` sets the number of assertions the test expects to run.  This count is checked when `done()` is called at the end of the test, and the test failed if there were missing/extra assertions.